Kickstarter-iOS/Info.plist

```diff
@@ -31,6 +31,7 @@
 		<dict>
 			<key>CFBundleURLSchemes</key>
 			<array>
+				<string>appcenter-$(APPCENTER_$(APPCENTER_SCHEME_NAME)_API_TOKEN)</string>
```

Comment:
@Scollaco @ifbarrera This accesses the environment var for the API token based on an environment var for this configuration.

So for our alpha target the concatenated environment var is `APPCENTER_ALPHA_API_TOKEN` and in this var name `ALPHA` is a variable based on `APPCENTER_SCHEME_NAME` which is a user-defined variable in our build settings.

Reply:
[These docs](https://docs.microsoft.com/en-us/appcenter/sdk/distribute/ios#23-modify-the-projects-infoplist) mention that we need to have the secret in the `Info.plist`